internal/pkg/utils/slice.go

```diff
@@ -19,7 +19,7 @@ func SortByName(slice interface{}) interface{} {
 		panic(fmt.Errorf("expected slice, given \"%T\"", slice))
 	}
 
-	// Sort by Name, and by String key if names are same
+	// Sort by Label, and by String key if names are same
 	value := reflect.ValueOf(slice)
```

Comment:
Ani tento komentar si asi nechcel zmenit.

Reply:
No bude to potrebne zmenit tu, iba zmenit v priklade `id -> value` a `Option` napr. na `Label`.
https://github.com/keboola/keboola-as-code/blob/5b25c5148781c49beabb527bc9ff17c38d5c527f/internal/pkg/cli/dialog/inputs_detail.go#L201-L208